src/quote/Quote.ts

```diff
@@ -40,6 +50,14 @@ export function validateQuoteSchema(
   return !!result
 }
 
+/**
+ * Builds a [[Quote]] object, from a simple object with the same properties.
+ *
+ * @param deserializedQuote The bae object which is used to create the attester signed [[Quote]] object.
+ *
```

Comment:
I would add to the description that this function does not only "clone" the quote object but also some checking. Also, the documentation is missing `@throws` for both errors.

Reply:
Yes, we currently don't use `@throws`, which could be useful. 

Something to discuss, at the moment I will not add.